tox.ini

```
@@ -47,3 +34,17 @@ deps =
 commands =
     coverage run -m pytest --basetemp={envtmpdir} {posargs}
     coverage report --fail-under 80 --omit .tox/*,unittests/*
+
+[testenv:dev]
+# the dev environment contains everything you need to start developing on your local machine.
+deps =
+    {[testenv:tests]deps}
+    {[testenv:linting]deps}
+    {[testenv:coverage]deps}
+    black
+    pip-tools
+    pre-commit
+commands =
+    python -m pip install --upgrade pip
+    pip-compile requirements.in
+    pip install -r requirements.txt
```

Das Zeilenende wird pre-commit in Zukunft fixen ;)
```suggestion
    pip install -r requirements.txt
    
```